src/main/java/racing/domain/Car.java

```diff
@@ -1,28 +1,65 @@
 package racing.domain;
+
+import java.util.Objects;
+
 /**
- * 자동차
- * 1. 전진 & 멈추기
- * 2. 현재 위치 반환
- * */
-public class Car {
-  private int position;
-  public Car(int position) {
+ * 자동차 1. 전진 & 멈추기 2. 현재 위치 반환
+ *
+ */
+public class Car implements Comparable<Car> {
+
+  private static final int MOVE_BORDER = 4;
+  private final CarName name;
+  private Position position;
+
+  public Car(String name) {
+    this(new CarName(name));
+  }
+
+  public Car(CarName name) {
+    this(name, new Position(0));
+  }
+
+  public Car(CarName name, Position position) {
+    this.name = name;
     this.position = position;
   }
 
-  public boolean move(int value){
-    if(!isMove(value)){
+  public void move(int value) {
+    if (value < MOVE_BORDER) {
+      return;
+    }
+    this.position.move(1);
```

Comment:
```suggestion
    this.position.increase();
```
외부에서 값을 전달하기보다 1 증가하는 부분을 위와 같이 추상화하도록 의식적인 연습을 해본다.